lib/lcset.gd

```diff
@@ -0,0 +1,51 @@
+##############################################################################
+##
+#W  lcset.gd                  GAP4 package `Utils'               Chris Wensley
+##
+#Y  Copyright (C) 2015-2022, The GAP Group 
+
+#############################################################################
+##
+#C  IsLeftCosetObj( <obj> ) . . . . . . . . . . . . . . coset of the form gU
+#R  IsLeftCosetDefaultRep
+#V  IsLeftCosetFamily 
+#T  IsLeftCosetType 
+#P  IsLeftCoset 
+##
+DeclareCategory( "IsLeftCosetObj", IsDomain and IsExternalOrbit ); 
+DeclareRepresentation( "IsLeftCosetDefaultRep",
+    IsComponentObjectRep and IsAttributeStoringRep and IsLeftCosetObj, 
+    [ "Representative", "ActingDomain" ] ); 
+BindGlobal( "IsLeftCosetFamily", NewFamily( "IsLeftCosetFamily", 
+            IsMultiplicativeElementWithInverse ) ); 
+BindGlobal( "IsLeftCosetType", 
+            NewType( IsLeftCosetFamily, IsLeftCosetDefaultRep ) ); 
+DeclareProperty( "IsLeftCoset", IsLeftCosetObj ); 
```

Comment:
I don't understand what the purpose of this property is. Do you expect there to be `IsLeftCosetObj` which are not left cosets?

Reply:
Done.